examples/step-45/doc/results.dox

```diff
@@ -1,155 +1,10 @@
 <h1>Results</h1>
 
-The textual output the program generates is not very surprising. It just
-prints out the usual information on degrees of freedom and active cells,
-in much the same way as step-3 did:
+The created output not very surprising. We simply see that the solution is
+periodic with respect to the left and lower boundary:
 
-@code
-Number of active cells: 1024
-Degrees of freedom: 1089
-@endcode
+<img src="http://www.dealii.org/images/steps/developer/step-45_periodic.png" alt="">
```

Comment:
What you plot in the right panel is actually the "speed", not the "velocity", I believe. Velocity is a vector, speed is its magnitude.